src/Models/LaunchesModel.js

```diff
@@ -62,35 +63,51 @@ export default class LaunchesModel {
   };
 
   @action
-  toggleNotifications = async () => {
-    this.notifications.enabled = !this.notifications.enabled;
-    this.storeNotificationSettings();
-    if (this.notifications.enabled) {
-      PushNotificationIOS.requestPermissions();
-    } else {
-      PushNotificationIOS.cancelAllLocalNotifications();
-      PushNotificationIOS.removeAllDeliveredNotifications();
-    }
-  };
+  toggleNotifications = Platform.select({
+    ios: async () => {
+      this.notifications.enabled = !this.notifications.enabled;
+      this.storeNotificationSettings();
+      if (this.notifications.enabled) {
+        PushNotificationIOS.requestPermissions();
+      } else {
+        PushNotificationIOS.cancelAllLocalNotifications();
+        PushNotificationIOS.removeAllDeliveredNotifications();
+      }
+    },
+    android: () => console.log("toggleNotifications")
```

Comment:
I guess asking for notifications permissions won't be a thing on Android. Although we don't need to request permission, we still need to enable the toggle that allows the user to toggle the notifications if he wants to.

With your changes, If the user tries to toggle notifications on Android, the code will just fire a `console.log`, and since you kept the `if (this.notifications.enabled) {...}` in the `scheduleNotification` method, the state of the `notifications.enabled` will never change (and the toggle won't move).

I recommend removing the `console.log` and keeping the toggle logic on Android: 

```suggestion
    android: () => {
        this.notifications.enabled = !this.notifications.enabled;
    }
```

Reply:
Definitely shouldn't have overlooked that, anyway should be good to go now!